src/asmcnc/apps/drywall_cutter_app/widget_drywall_shape_display.py

```diff
@@ -500,13 +500,13 @@ def select_shape(self, shape, rotation, swap_lengths=False):
 
         if shape == 'geberit':
             if rotation == 'horizontal':
-                self.place_widget(self.x_datum_label, (407, 46))
-                self.place_widget(self.y_datum_label, (416, 125))
+                self.place_widget(self.x_datum_label, (358, 50))
+                self.place_widget(self.y_datum_label, (400, 207))
```

Comment:
Maybe a question for Marc, but we need to scale the Shapes for C10 & C7, so may be worth implementing that now. @Masc999

Reply:
We will not need this for the gerberit shape, but for all others as Shapes will be available both on C7 and C10.
But I think we will do this in a separate ticket, as we want the DWT changes to go out with the 2.9.1 (hotfix) release.